cloud/sample.env

```diff
@@ -54,3 +54,5 @@ COMPOSE_PROFILES=dev
 # use the local one instead. Only use when developing @transitive-robotics
 # capabilities locally.
 # TR_REGISTRY_IS_LOCAL=true
+
+HYPERDX_API_KEY=api_key_for_opentelemetry_collector
```

Comment:
Were you able to get this to work? For me it still shows a randomly generated key even when I set this. But it's possible I'm doing something wrong.